content/cli/install.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: bt_wiki
+title: install
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 100
+---
+
+The `cfy install` command is used to install an application using a Cloudify manager without going through the process of uploading a blueprint, creating a deployment and executing a workflow.
```

Comment:
it is somewhat confusing mentioning the you don't go through the process. While you don't *manually* go through the process, this is exactly the process that is automated for you